arm/Microsoft.Automation/automationAccounts/softwareUpdateConfigurations/deploy.bicep

```diff
@@ -174,24 +174,32 @@ param nextRunOffsetMinutes int = 0
 @description('Optional. The schedules description.')
 param scheduleDescription string = ''
 
-@description('Generated. Do not touch. Is used to provide the base time for time comparrison for startTime. If startTime is specified in HH:MM format, baseTime is used to check if the provided startTime has passed, adding one day before setting the deployment schedule.')
+@description('Generated. Do not touch. Is used to provide the base time for time comparison for startTime. If startTime is specified in HH:MM format, baseTime is used to check if the provided startTime has passed, adding one day before setting the deployment schedule.')
 param baseTime string = utcNow('u')
 
 @description('Optional. Customer Usage Attribution id (GUID). This GUID must be previously registered')
 param cuaId string = ''
 
 var updateClassifications_var = '${replace(replace(replace(replace(string(updateClassifications), ',', ', '), '[', ''), ']', ''), '"', '')}'
-var timeLimit = dateTimeAdd(baseTime, 'PT5M', 'u')
-var providedStartTime = dateTimeAdd(startTime, 'PT0S', 'u')
-var startTime_var = ((providedStartTime > timeLimit) ? providedStartTime : dateTimeAdd(providedStartTime, 'P1D', 'u'))
+
+// var timeLimit = dateTimeAdd(baseTime, 'PT5M', 'u')
+
+// var providedStartTime = dateTimeAdd(startTime, 'PT0S', 'u')
+
+// var startTime_var = providedStartTime > timeLimit ? providedStartTime : dateTimeAdd(providedStartTime, 'P1D', 'u')
 
 module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
   name: 'pid-${cuaId}'
   params: {}
 }
 
+resource softwareUpdateConfiguration_automationAccount 'Microsoft.Automation/automationAccounts@2020-01-13-preview' existing = {
+  name: automationAccountName
+}
+
 resource softwareUpdateConfiguration 'Microsoft.Automation/automationAccounts/softwareUpdateConfigurations@2019-06-01' = {
-  name: '${automationAccountName}/${deploymentScheduleName}'
+  name: name
+  parent: softwareUpdateConfiguration_automationAccount
```

Comment:
I know you're still in a draft mode, but as I'm just going through the entire repo on my search of non-aligned child-resources -> here is one :D